contrib/devtools/gen-manpages.py

```diff
@@ -62,6 +62,10 @@
     # Copyright is the same for all binaries, so just use the first.
     footer.write('[COPYRIGHT]\n')
     footer.write('\n'.join(versions[0][2]).strip())
+    # Create SEE ALSO section
+    footer.write('\n[SEE ALSO]\n')
+    footer.write(', '.join(s.rpartition('/')[2] + '(1)' for s in BINARIES))
```

Comment:
Could use `os.path.basename` it's slightly more readable and will always pick the last path item.